src/SFA.DAS.EmployerIncentives.Api/HealthChecks/CommitmentsApiHealthCheck.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using SFA.DAS.EmployerIncentives.Interfaces;
+
+namespace SFA.DAS.EmployerIncentives.Api.HealthChecks
+{
+    public class CommitmentsApiHealthCheck : IHealthCheck
```

Comment:
This isn't used and can be deleted - CommitmentsHealthCheck is what is registered in Startup

Reply:
sorry missed it